resources/config/tasks.yml

```diff
@@ -281,3 +281,13 @@ services:
             - '@linter.yamllint'
         tags:
             - {name: grumphp.task, config: yamllint}
+
+    task.phpversion:
+        class: GrumPHP\Task\PhpVersion
+        arguments:
+            - '@config'
+            -
+              5.6: '2016-12-31 23:59:59'
```

Comment:
Two questions:

- Should we add old versions for a complete overview?
- Should we use the active support or security support as an end-date? Security support is still support ... :)